Formula/bastet.rb

```diff
@@ -33,4 +33,10 @@ def install
     bin.install "bastet"
     man6.install "bastet.6"
   end
+
+  test do
+    # Installation of Bastet creates an executable file called Bastet
```

Comment:
Could you try running and afterwards killing the program? Something like this might work

```ruby
    begin
      pid = fork do
        exec bin/"basted"
      end
      sleep 3
    ensure
      Process.kill("TERM", pid)
      Process.wait(pid)
    end
```

Reply:
Hi! I've made the changes, but I still don't get what we're testing when we try running and then killing the program.
I'm confused about what exactly the test is doing here? @SMillerDev

Reply:
I'm fairly confident that brew moves files around correctly. We don't have to test that, I think there are even tests for that in homebrew/brew. The point of tests in homebrew-core is to see if the configuration and compilation is correct. That will require running the program.

Reply:
Got it! :)

Reply:
I think this PR is ready too. Let me know if some changes are required 😄  @SMillerDev

Reply:
Can you remove the comment (and make sure there's still just a single commit after)

Reply:
On it. 💯

Reply:
Done. Waiting for tests to finish. :)